### PR TITLE
Add customizable Standard wish pools and 10-pull costs

### PR DESCRIPTION
**Pull request by SvardXI**
## Description
Added the ability for users to change what items they want on the Standard wish for each of their banners, as well as the ability to to adjust the cost of 10-pulls for discounted banners.

Note: If the field(s) in the JSON is empty for the Standard pools, the server will still use mHY's official Standard pool. Additionally, while the cost of a 10-pull will show in the game's UI, it will only do it before making a 10-pull. The game will still check for (and consume) the adjusted cost.

## Type of changes

- [ ] Bug fix
- [ ] New feature 
- [x] Enhancement
- [ ] Documentation

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] My pull request is unique and no other pull requests have been opened for these changes
- [x] I have read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md)
- [x] I am responsible for any copyright issues with my code if it occurs in the future.

### CODE REVIEW

**Comment by SvardXI** [comment]
Recently noticed one issue with my commit--

I used a source file that was a about a day old. As such, line 129 needs to be replaced:
```
	- String record = "https://"
	+ String record = "http" + (Grasscutter.getConfig().getDispatchOptions().FrontHTTPS ? "s" : "") + "://"
```

File: src/main/java/emu/grasscutter/game/gacha/GachaBanner.java [hunk 1]
```diff
@@ -96,7 +126,7 @@ public GachaInfo toProto() {
 		return toProto("");
 	}
 	public GachaInfo toProto(String sessionKey) {
-		String record = "http" + (Grasscutter.getConfig().getDispatchOptions().FrontHTTPS ? "s" : "") + "://"
+		String record = "https://"
```
Comment (ghost) [inline src/main/java/emu/grasscutter/game/gacha/GachaBanner.java]: Please do not revert changes made in other PRs. Ensure your fork is up-to-date or be very careful when merging conflicts.

**Review by ghost — CHANGES_REQUESTED** [review]

File: src/main/java/emu/grasscutter/game/gacha/GachaBanner.java (hunk 1 above)
Comment (HotaruYS) [reply src/main/java/emu/grasscutter/game/gacha/GachaBanner.java:129]: https://github.com/Grasscutters/Grasscutter/pull/649

**Comment by ghost** [comment]
Conflicts with changes in https://github.com/Grasscutters/Grasscutter/pull/639

---

>I used a source file that was a about a day old. As such, line 129 needs to be replaced:

You will need to do it yourself, otherwise someone would have to make another PR to fix it.

**Comment by ghost** [comment]
Thank you for wanting to contribute to our project, but i will be closing this PR as these changes were added in #639

**Pull request closed without merging**